src/e2e/app/cli/common/src/kamu_api_server_client.rs

```diff
@@ -76,6 +80,29 @@ impl KamuApiServerClient {
         Ok(())
     }
 
+    pub async fn set_system_time(&self, t: DateTime<Utc>) {
+        let endpoint = self.server_base_url.join("e2e/system_time").unwrap();
```

Comment:
Since the number of REST endpoints is growing, we should be looking for a way to enumerate them or have a list of constants, rather than copypasting the suffixes in the code.

Reply:
Yes, I think about this periodically when I'm forced to use hardcode in tests. After the swagger integration that defines endpoint usage via the `#[utoipa::path()]` macro, I have no idea how to do it more or less nicely yet. 

Q: Maybe you have some ideas?